tinygrad/features/search.py

```diff
@@ -46,7 +46,7 @@ def time_linearizer(lin:Linearizer, rawbufs:List[RawBuffer], allow_test_size=Tru
 
     # TODO: this is copied from prg.__call__
     global_size, local_size = prg.launch_dims(var_vals)
-    if global_size is not None and local_size is None and all_int(tuple(prg.global_size)): # type: ignore[arg-type]
+    if global_size is not None and local_size is None and all_int(tuple(cast(Iterable[Any], prg.global_size))):
```

Comment:
Don't do things like this. This is worse than a type: ignore

Reply:
I'll look for a better solution